crates/kitsune_p2p/mdns/Cargo.toml

```diff
@@ -22,7 +22,7 @@ path = "examples/discover.rs"
 [dependencies]
 libmdns = "=0.7.4"
 mdns = "=3.0.0"
-futures = "0.3"
+futures-util = "0.3"
```

Comment:
This was an accidental change on another PR for removing workspace dependencies and was preventing an example from building